packages/core/README.md

```diff
@@ -21,16 +21,18 @@ initializeAgent({
 
 The agent requires a configuration parameter.
 
-| Property                | Description                                                             | Type                | Optional | Default Value  |
-| ----------------------- | ----------------------------------------------------------------------- | ------------------- | -------- | -------------- |
-| `app`                   | Application metadata                                                    | `App`               | Y        | `undefined`    |
-| `globalObjectKey`       | String that should be used when defining the agent on the global object | `string`            | Y        | `grafanaAgent` |
-| `instrumentations`      | Array of instrumentations that should be ran                            | `Instrumentation[]` | N        | `[]`           |
-| `metas`                 | Array of metas that should be logged                                    | `MetaItem[]`        | Y        | `[]`           |
-| `preventGlobalExposure` | Flag for toggling the definition on the global object                   | `boolean`           | Y        | `false`        |
-| `session`               | Session metadata                                                        | `Session`           | Y        | `undefined`    |
-| `transports`            | Array of transports that should be used                                 | `Transport[]`       | Y        | `[]`           |
-| `user`                  | User metadata                                                           | `User`              | Y        | `undefined`    |
+| Property                | Description                                                             | Type                                       | Optional | Default Value  |
+| ----------------------- | ----------------------------------------------------------------------- | ------------------------------------------ | -------- | -------------- |
+| `app`                   | Application metadata                                                    | `App`                                      | Y        | `undefined`    |
+| `beforeSend`            | Hook to modify or discard an event before it is sent over the wire      | `(TransportItem) => TransportItem \| null` | Y        | `undefined`    |
+| `globalObjectKey`       | String that should be used when defining the agent on the global object | `string`                                   | Y        | `grafanaAgent` |
+| `ignoreErrors`          | Error message patterns for errors that should be ignored                | `Array<string \| RegExp>`                  | Y        | `[]`           |
```

Comment:
```suggestion
| `ignoreErrors`          | Error message patterns for errors that should be ignored                | `Patterns`                  | Y        | `[]`           |
```

Reply:
Hm, not sure about aliasing such simple types.. Where do we define them then? below the table?

Reply:
We can point to the definition in the code using a link 🙂

Reply:
ok, makes sense
- [ ] point to type in code

Reply:
I changed the type, but we'll have to link in a separate PR. This code is not in main yet

Reply:
👍